mainnet1/genesis.json

```diff
@@ -363,7 +363,15 @@
       "spam.protection.max.proposals": "3",
       "spam.protection.proposal.min.tokens": "200000000000000000000000",
       "spam.protection.max.delegations": "360",
-      "spam.protection.delegation.min.tokens": "1"
+      "spam.protection.delegation.min.tokens": "1",
+      "governance.proposal.freeform.minProposerBalance": "1000000000000000000",
+      "governance.proposal.freeform.minVoterBalance": "1000000000000000000",
+      "governance.proposal.freeform.minClose": "48h0m0s",
```

Comment:
I'd be tempted to have this something like 3 days, maybe a whole week to allow time for discussion.